src/capi-single-paid/index.js

```diff
@@ -0,0 +1,73 @@
+import config from '../_shared/js/config';
+import { enableToggles } from '../_shared/js/ui.js';
+import { write } from '../_shared/js/dom';
+import { getIframeId, getWebfonts, resizeIframeHeight } from '../_shared/js/messages';
+import { portify } from '../_shared/js/dev';
+
+let container = document.getElementsByClassName('adverts__body')[0];
+let params = new URLSearchParams();
+let keywords = '[%SeriesUrl%]';
+let customUrl = '[%CustomUrl%]';
+
+if (customUrl !== '') {
+  params.append('t', customUrl);
+} else {
+  params.append('k', keywords);
+}
+
+const GLABS_EDITION = {
+	default: 'https://theguardian.com/guardian-labs',
```

Comment:
You can strip the `https://theguardian.com` part since I added it to the `BASE` element. All URIs that are not fully-qualified will open on the guardian.